### PR TITLE
[dhctl] Change color, fix empty host connection attempts

### PR DESCRIPTION
**Pull request by nabokihms**
Signed-off-by: m.nabokikh <maksim.nabokikh@flant.com>

## Description
* Changes font colour for bold blocks from white to normal
* Fail if there is empty host for ssh connection

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: dhctl
type: fix
summary: Changes font colour for bold blocks from white to normal
impact_level: low
---
section: dhctl
type: fix
summary: Fail if there is empty host for ssh connection
impact_level: default
```

### CODE REVIEW

File: dhctl/pkg/system/ssh/frontend/waiting.go [hunk 1]
```diff
@@ -39,6 +39,9 @@ func (c *Check) WithDelaySeconds(seconds int) *Check {
 }
 
 func (c *Check) AwaitAvailability() error {
+	if c.Session.Host() == "" {
+		return fmt.Errorf("empty host for connection received")
```
Comment (name212) [inline dhctl/pkg/system/ssh/frontend/waiting.go:43]:
```suggestion
		return fmt.Errorf("Empty host for connection received")
```

**Review by name212 — CHANGES_REQUESTED** [review]
One suggest and plese run e2e

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @nabokihms. I've started the workflow for label 'e2e/run/gcp'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @nabokihms. I've started the workflow for label 'e2e/run/azure'...

**Review by name212 — APPROVED** [review]

**Pull request merged**